data_steward/analytics/cdr_ops/raw_rdr_export_qc.py

```diff
@@ -1250,39 +1069,66 @@
 all_queries = '\nUNION ALL\n'.join(queries)
 execute(client, f'{src_ids_table}\n{all_queries}')
 
+
 # # Check Wear Consent Counts
+# This query checks the count of wear consent observations. If the number of observations is not decreasing this check will pass.
 #
-# `Wear_consent` and `wear_consent_ptsc` records should be seen in the export.
-#
-# Results expectations: The result should be roughly 16 rows. Differences here most likely aren't an issue.
-#
-# **Visual check:** <br>
-# * **PASS**   The result **includes** observation_source_value: `resultsconsent_wear` <br>
-# * **FAIL**   The result **does not include** observation_source_value: `resultsconsent_wear`.  If this row does not exist, confirm the finding, and report to RDR. These records are required for proper suppression of wear fitbit records.
+# **If this check fails** Investigate why the number of observations have decreased. These data are important for the creation of the wear_study table and therefore data suppression
 
+# +
 # Get counts of wear_consent records
 query = JINJA_ENV.from_string("""
+
 SELECT
-  observation_source_value,
-  COUNT(*) AS n
-FROM
-  `{{project_id}}.{{new_rdr}}.observation` o
-  LEFT JOIN   `{{project_id}}.{{new_rdr}}.survey_conduct` sc
-  ON sc.survey_conduct_id = o.questionnaire_response_id
-WHERE sc.survey_concept_id IN (2100000011,2100000012) -- captures questions asked in multiple surveys --
-OR LOWER(observation_source_value) IN UNNEST ({{wear_codes}}) -- captures those that might be missing from survey_conduct --
-GROUP BY 1
+  curr.observation_source_value AS concept
+ ,prev.row_count AS _{{old_rdr}}
+ ,curr.row_count AS _{{new_rdr}}
+ ,(curr.row_count - prev.row_count) row_diff
+FROM (SELECT DISTINCT observation_source_value, COUNT(*) as row_count
+    FROM `{{project_id}}.{{new_rdr}}.observation` o
+    WHERE observation_source_value = 'resultsconsent_wear'
+    GROUP BY 1) curr
+JOIN (SELECT DISTINCT observation_source_value, COUNT(*) row_count 
+    FROM `{{project_id}}.{{old_rdr}}.observation` o
+    WHERE observation_source_value = 'resultsconsent_wear'
+    GROUP BY 1) prev
+USING (observation_source_value)
+GROUP BY 1,2,3
```

Comment:
Now that expectations for wear records are known. Updated the query to only show possible issues.